app/angular/src/client/preview/decorateStory.ts

```diff
@@ -1,31 +1,23 @@
-import { DecoratorFunction, StoryContext, StoryFn } from '@storybook/addons';
+import { DecoratorFunction, LegacyStoryFn, StoryContext } from '@storybook/csf';
+import { sanitizeStoryContextUpdate } from '@storybook/store';
 import { computesTemplateFromComponent } from './angular-beta/ComputesTemplateFromComponent';
-import { StoryFnAngularReturnType } from './types';
 
-const defaultContext: StoryContext = {
-  id: 'unspecified',
-  name: 'unspecified',
-  kind: 'unspecified',
-  parameters: {},
-  args: {},
-  argTypes: {},
-  globals: {},
-};
+import { AngularFramework } from './types-6-0';
 
 export default function decorateStory(
-  mainStoryFn: StoryFn<StoryFnAngularReturnType>,
-  decorators: DecoratorFunction<StoryFnAngularReturnType>[]
-): StoryFn<StoryFnAngularReturnType> {
+  mainStoryFn: LegacyStoryFn<AngularFramework>,
```

Comment:
how to get rid of the LegacyStoryFn ? ( in the future )

it's not the same guy as  ProjectAnnotations<...>['applyDecorators'];

only for my personal understanding 😄 🙏